### PR TITLE
Feature/user section v2

### PR DESCRIPTION
**Pull request by JesmoDev**

### CODE REVIEW

File: src/backoffice/sections/users/views/users/editor-view-users-invite.element.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
 import { customElement, query, state } from 'lit/decorators.js';
 import { UmbModalLayoutElement } from '../../../../../core/services/modal/layouts/modal-layout.element';
 import { UmbUserStore } from '../../../../../core/stores/user/user.store';
```
Comment (iOvergaard) [inline src/backoffice/sections/users/views/users/editor-view-users-invite.element.ts:5]: Consider using shortcut imports

File: src/backoffice/sections/users/views/users/editor-view-users-invite.element.ts [hunk 2]
```diff
@@ -72,12 +75,17 @@ export class UmbEditorViewUsersInviteElement extends UmbContextConsumerMixin(Umb
 
 		const formData = new FormData(form);
 
+		console.log('formData', formData);
```
Comment (iOvergaard) [inline src/backoffice/sections/users/views/users/editor-view-users-invite.element.ts:78]: console.log

**Review by iOvergaard — DISMISSED** [review]
Generally looks good for a work-in-progress feature. I marked some imports that need to be looked after - they should probably have a cleanup in most of the files when/if you decide to make new shortcuts.

File: src/backoffice/editors/user/editor-user.element.ts [hunk 3]
```diff
@@ -183,56 +169,93 @@ export class UmbEditorUserElement extends UmbContextProviderMixin(UmbContextCons
...
+				return html`
+					<uui-ref-node name=${node}>
+						<uui-icon slot="icon" name="folder"></uui-icon>
+					</uui-ref-node>
+				`;
+			}
+		);
+	}
+
 	private renderLeftColumn() {
```
Comment (nielslyngsoe) [inline src/backoffice/editors/user/editor-user.element.ts:211]: Let's add "_" to this name to ensure it is private at runtime.

**Review by nielslyngsoe — APPROVED** [review]

**Pull request merged**